### PR TITLE
Adds hardcoded version and adds github workflow files

### PR DESCRIPTION
**Pull request by saltiyazan**
- Adds the `version` keyword to the snapcraft.yaml file and uses a hardcoded version.
- Updates the repository to use the shared Github Actions. (Snapcrafters/ci)

### CODE REVIEW

**Comment by saltiyazan** [comment]
Hi @merlijn-sebrechts 
Here is my first PR to Snapcrafters, I followed the contributing guide to add the workflows.
Could you please take a look?

File: .github/workflows/release-to-candidate.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    needs: call-for-testing
+    environment: "Candidate Branch"
+    runs-on: ubuntu-latest
+    steps:
+      - name: 📸 Gather screenshots
+        uses: snapcrafters/ci/get-screenshots@main
+        with:
+          issue-number: ${{ needs.call-for-testing.outputs.issue-number }}
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          screenshots-token: ${{ secrets.SNAPCRAFTERS_BOT_COMMIT }}
```
Comment (jnsgruk) [inline .github/workflows/release-to-candidate.yml]: I don't think this is likely to be useful in a CLI application - I think it can be removed :)

**Review by jnsgruk — DISMISSED** [review]
LGTM - probably worth removing the `get-screenshot` action before we merge. Once you've got that sorted.

I'll add the credentials to the repo now.

**Comment by jnsgruk** [comment]
Could you please also add a README using the template shown in https://github.com/snapcrafters/discord?

File: snap/snapcraft.yaml [hunk 2]
```diff
@@ -3,11 +3,15 @@ adopt-info: get-iplayer
...
     Command Line tool for downloading tv and radio programmes from the last 30 days from the BBC iPlayer.
+version: "3.35"
 base: core22
 grade: stable 
 confinement: strict
 assumes:
   - command-chain
+architectures:
+  - amd64
+  - arm64
```
Comment (jnsgruk) [inline snap/snapcraft.yaml:14]:
```suggestion
architectures:
  - amd64
  - arm64
  - armhf
  - ppc64el
  - s390x
```

The snap currently has releases across all of these architectures :) I don't think anyone is actually using the `ppc64el` or `s390x` releases (according to the metrics), but we should probably leave `armhf` until it causes us problems.

File: .github/workflows/release-to-candidate.yml (hunk 1 above)
Comment (saltiyazan) [reply .github/workflows/release-to-candidate.yml]: ✅

File: snap/snapcraft.yaml (hunk 2 above)
Comment (saltiyazan) [reply snap/snapcraft.yaml:14]: ✅

**Comment by saltiyazan** [comment]
> Could you please also add a README using the template shown in https://github.com/snapcrafters/discord?

I added it, I'm not fully sure about the maintainers and license sections (I added the LICENSE file too) :)

File: README.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+</h1>
+
+<p align="center"><b>This is the snap for get_iplayer</b>. It is a community-maintained package to easily install get_iplayer on Ubuntu, Fedora, Debian and other major Linux distributions. It is available in the Snap Store, Ubuntu Software, and a number of other applications.</p>
+
+<p align="center"><i>"get_iplayer is a utility for downloading TV and radio programmes from BBC iPlayer and BBC Sounds"</i></p>
+
+<p align="center">
+<a href="https://snapcraft.io/get-iplayer"><img src="https://snapcraft.io/get-iplayer/badge.svg" alt="Snap Status"></a>
+<a href="https://github.com/snapcrafters/get-iplayer/actions/workflows/sync-version-with-upstream.yml"><img src="https://github.com/snapcrafters/get-iplayer/actions/workflows/sync-version-with-upstream.yml/badge.svg"></a>
+<a href="https://github.com/snapcrafters/get-iplayer/actions/workflows/release-to-candidate.yaml"><img src="https://github.com/snapcrafters/get-iplayer/actions/workflows/release-to-candidate.yaml/badge.svg"></a>
```
Comment (jnsgruk) [inline README.md]:
```suggestion
<a href="https://github.com/snapcrafters/get-iplayer/actions/workflows/release-to-candidate.yml"><img src="https://github.com/snapcrafters/get-iplayer/actions/workflows/release-to-candidate.yml/badge.svg"></a>
```

**Review by jnsgruk — DISMISSED** [review]

**Review by jnsgruk — APPROVED** [review]

**Comment by jnsgruk** [comment]
Thanks @saltiyazan!

**Pull request merged**